### PR TITLE
TAI pronunciation guide

### PR DESCRIPTION
**Pull request by talha-ahsan**
RwR is still having some changes so I'm waiting for a thumbs up before porting those over

### CODE REVIEW

File: v2/cards/bahia_bands.json [hunk 1]
```diff
@@ -7,6 +7,8 @@
   "id": "bahia_bands",
   "influence_cost": 1,
   "is_unique": false,
+  "pronunciation_approx": "BYE-yah",
```
Comment (plural) [inline v2/cards/bahia_bands.json:10]: any reason this doesn't have bands in here as well?

Comment (plural) [reply v2/cards/bahia_bands.json:10]: a similar question for other cards.  Why does the pronunciation only have part of the title?

If that's the convention, cool cool, but i'm curious.

**Review by plural — APPROVED** [review]

File: v2/cards/bahia_bands.json (hunk 1 above)
Comment (talha-ahsan) [reply v2/cards/bahia_bands.json:10]: This is more or less the convention AFAIK. I'm assuming English words with known pronunciation by native English speakers were omitted. They're not in the guide, and I don't trust myself to generate proper IPA for them on my own

Comment (plural) [reply v2/cards/bahia_bands.json:10]: that's what I expected.  Thanks for confirming.  :)

**Review by plural — APPROVED** [review]

**Pull request merged**